Packs/FidelisEndpoint/Integrations/FidelisEndpoint/FidelisEndpoint.yml

```diff
@@ -0,0 +1,1619 @@
+category: Endpoint
+commonfields:
+  id: Fidelis EDR
+  version: -1
+configuration:
+- display: Server URL (e.g. https://abcde.fideliscloud.com/)
+  name: url
+  required: true
+  type: 0
+- display: Username
+  name: credentials
+  required: true
+  type: 9
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: Fetch incidents query
+  name: isFetch
+  required: false
+  type: 8
+- defaultvalue: 3 days
+  display: First fetch timestamp (<number> <time unit>, e.g., 12 hours, 7 days, 3
+    months, 1 year)
+  name: fetch_time
+  required: false
+  type: 0
+- display: severity
+  name: severity
+  required: false
+  type: 0
```

Comment:
do we know if the severity value can be one of `High`, `Medium` and `Low`, set this param to be multi select